site/docs/components/SwapComponents.tsx

```diff
@@ -74,22 +74,6 @@ export default function SwapComponents() {
 
   const swappableTokens = [DEGENToken, ETHToken, USDCToken, WETHToken];
 
-  const onSubmit = useCallback(
-    async (swapTransaction: BuildSwapTransaction) => {
-      const { transaction } = swapTransaction;
-      console.log('Prepared swapTransaction:', transaction);
-      // Uncomment the following line to send the transaction
-      // for Localhost testing
-      // const tx = await sendTransaction({
-      //   to: transaction.to,
-      //   value: transaction.value,
-      //   data: transaction.data,
-      // });
-      setPreparedTransaction(transaction);
-    },
-    [],
-  );
-
```

Comment:
this we still want this to see this in the console

Reply:
hmm... maybe we don't want it to disable it and just show the admonition + show the callback response in console log

Reply:
We are displaying this when the user clicks submit

<img width="1133" alt="Screenshot 2024-06-17 at 12 35 25 PM" src="https://github.com/coinbase/onchainkit/assets/132851666/109af549-f92a-4b05-b5cd-6245ac365ddc">

Reply:
removed per legal